.github/workflows/test.yml

```diff
@@ -0,0 +1,76 @@
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - 'releases/*'
+
+jobs:
+  build_test_job:
+    name: 'Build and test job'
+    runs-on: ${{ matrix.os }}
+    strategy:
+        matrix:
+          os: [windows-latest, ubuntu-latest, macos-latest]
+    steps:
+
+    - name: 'Checking out repo code'
+      uses: actions/checkout@v1
+
+    - name: Extract branch name
+      id: extract_branch
+      run: |
+        echo "##[set-output name=branchname;]$(echo ${GITHUB_REF##*/})"
+
+    - name: 'Validate build'
```

Comment:
This would fail in case of `release/v1` branch because we don't check in devDependencies node_modules. How do you plan on handling that?

Reply:
Updated for release branch condition. Can you please have look @ds-ms ?